event-log-service-webapp/pom.xml

```diff
@@ -36,13 +36,11 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-api</artifactId>
             <version>${log4j.version}</version>
-            <scope>provided</scope>
```

Comment:
@rohit-yawalkar @deepthi-mantena do we have to package the log4j as part of this jar ? any specific reason for it ?

Reply:
No Bindu, its building fine